### DISCUSSION TITLE
How to print using SuperSlicer, and the differences between the included print settings.

### DISCUSSION THREAD

**Discussion by bassamanator** [bassamanator/Sovol-SV06-firmware#14] [Announcements]
[![IMAGE ALT TEXT HERE](http://img.youtube.com/vi/LZN7IVf4TWs/0.jpg)](http://www.youtube.com/watch?v=LZN7IVf4TWs)

Full differences between `Standard` and `45 degree` print profiles can be found [here](https://github.com/AndrewEllis93/Ellis-SuperSlicer-Profiles/tree/master/SuperSlicer). But basically, the 45 degree profile places the `seam` at the back.

**Comment by MihkelBaranov** [comment]
Thanks for this
